app/views/guests/show.html.erb

```diff
@@ -0,0 +1,17 @@
+<h1> <%= @guest.name %> </h1>
+
+<h3> Add a Room to <%=@guest.name%>: </h3>
+
+<%= form_with url:"/guests/#{@guest.id}/room_guests", method: :post, local: true do |form| %>  
+  <%= form.label "Room ID: " %>
+  <%= form.number_field :room_id %>
+  <%= form.submit "Submit"%>
+<% end %>
+<h3> <%=@guest.name%>'s Rooms: </h3>
+
+<% @guest.rooms.each do |room| %>
+  <p> Suite: <%= room.suite %> </p>
+  <p> Nightly Rate: <%= room.rate %> </p>
+  <p> Hotel: <%= room.hotel.name %> </p>
+  <br>
+<% end %>
```

Comment:
I already mentioned this in the `routes.rb` file, but the URL for the form isn't correct according to REST.

```ruby
<%= form_with url: "/guests/#{@guest.id}/rooms", method: :post, local: true do |f| %>
```